src/components/NewRuntimeModal.js

```diff
@@ -285,24 +286,32 @@ export const NewRuntimeModal = withModalDrawer({ width: 675 })(class NewRuntimeM
       })
     }
 
+    const customEnvVars = {
+      WORKSPACE_NAME: name,
```

Comment:
Thinking through if setting `WORKSPACE_NAME` and `WORKSPACE_BUCKET` EVs will cause any issues.

Currently runtimes can still be used across workspaces. Leo has logic to dynamically set `WORKSPACE_NAME` and `WORKSPACE_BUCKET` depending on the current welder storage link. That way your notebook runtime is always connected to the workspace it's being used from.

I think that dynamic-ev-setting should overwrite what's specified in `customEnvVars`. So I _think_ it should be okay to set it here. Once all workspaces are migrated to PPW we can remove the welder logic to dynamically set EVs (and maybe other simplifications too).

Does all that sound right? Maybe we could test using a notebook runtime from 2 workspaces, and make sure the EVs are still set correctly.

Reply:
Good call, I make two workspaces, created a notebook in each workspace, and then swapped between the two and ensure `printenv` behaved properly within a notebook.

It all seemed to work fine, even when I have both notebooks open side by side